packages/@expo/cli/src/prebuild/updatePackageJson.ts

```diff
@@ -66,9 +66,9 @@ export async function updatePackageJSONAsync(
  * 3. Update `package.json` `main`.
  *
  * @param projectRoot The root directory of the project.
- * @param props.templatePkg Template project package.json as JSON.
- * @param props.pkg Current package.json as JSON.
- * @param props.skipDependencyUpdate Array of dependencies to skip updating.
```

Comment:
I think these `props.x` notations were correct JSDoc parameters, why did you remove the `props.` prefix?

![image](https://github.com/expo/expo/assets/1203991/35aaf5ee-d6db-4edc-9c2c-cead6469a982)

Reply:
This notation only applies to named parameters, and if using destructure notation `options` is a correct default name, so TL/DR it could be `options.templatePkg` or `templatePkg`, but not `props.templatePkg` unless we name the parameter manually as `props`.

Also, we are using TypeDoc, which relies on a mix of JSDoc, TSDoc and their own standard, so we should in most cases refer to their docs instead:
* https://typedoc.org/tags/param/